cli/js/testing.ts

```diff
@@ -212,7 +212,9 @@ function reportToConsole(message: TestMessage): void {
       }
     }
     log(
-      `\ntest result: ${message.end.failed ? RED_FAILED : GREEN_OK}. ` +
+      `\n${isDoctest ? "doctest" : "test"} result: ${
```

Comment:
Nitpick: seems unnecessary to print `doctest` if `--docs` flag is passed. Does that follow Rust's output?